client/src/reducers/alertReducer.js

```diff
@@ -0,0 +1,44 @@
+import {SET_ALERT,REMOVE_ALERT} from '../constants/alertConstants';
+
+const initialState = {
+  loginErr:[],
+  registerErr:[],
+  loadUserErr:[],
+  otherErr:[]
+}
+
+const alertReducer = (state=initialState,action) => {
```

Comment:
I think it would be cleaner to just return the errors/success status and do both redirecting and error display in each component, so we could do away with all of the alert reducers :thinking: